src/hooks/useCountries.jsx

```diff
@@ -1,16 +1,20 @@
 'use client'
 
 import { useQuery} from '@tanstack/react-query'
-import fetchData from './fetchData'
+import fetchData from '../app/api/fetchData'
 
 const useCountries = (options) => {
   return useQuery({
     queryKey: ['countries'],
-    queryFn: async () => {
+    queryFn: () => {
```

Comment:
```suggestion
    queryFn: fetchData('/all', {filters: options?.filters ?? [])
```
Odbierasz cały obiekt `options`, który może mieć różne opcje, również te z RQ, bo niżej spreadujemy, dlatego nie wysyłajmy tam nic czego nie potrzebujemy. W TypeScripcie byłoby to tak otypowane i nie mógłbyś tak tego przekazać.

Reply:
Popatrz na suggested change. Damy radę to zrobić w taki sposób?